hax/hax/util.py

```diff
@@ -1278,6 +1278,32 @@ def update_pver_bc(self, data: ByteCountStats) -> None:
             LOG.debug('Setting bytecount stats in KV: %s:%s', key, value)
             self.kv.kv_put(key, value)
 
+    def update_bc_for_dg_category(self,
+                                  pver_items: Dict[str, PverState]) -> None:
+        '''
+        This function will update bytecount for subsequent dg state/category
+        which will reflect on cluster status.
+        '''
+        pool_ver = self.kv.kv_get('ioservices/', recurse=True)
```

Comment:
please add a comment stating the corresponding kv with an example.

Reply:
added